src/gretel_trainer/trainer.py

```diff
@@ -151,6 +151,17 @@ def generate(self, num_records: int = 500) -> pd.DataFrame:
         )
         return self.run.get_synthetic_data()
 
+    def get_sqs_score(self) -> float:
+        """Return the average SQS synthetic data quality score.
+
+        Requires the model has been trained.
+        """
+        scores = [
+            sqs["synthetic_data_quality_score"]["score"]
+            for sqs in self.run.get_sqs_information
+        ]
+        return sum(scores) / len(scores)
```

Comment:
We should probably cast this return to `int`. Our SQS scores are always integers, this could return a float